Source/PLATEAUTests/Tests/PLATEAUInstancedCityModelTest.cpp

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright © 2023 Ministry of Land, Infrastructure and Transport
+
+#include "FileHelpers.h"
+#include "PLATEAUAutomationTestBase.h"
+#include "PLATEAUCityModelLoader.h"
+#include "PLATEAUInstancedCityModel.h"
+#include "Kismet/GameplayStatics.h"
+
+
+IMPLEMENT_CUSTOM_SIMPLE_AUTOMATION_TEST(FPLATEAUInstancedCityModelTest, FPLATEAUAutomationTestBase, "PLATEAUTest.InstancedCityModel",
+                                        EAutomationTestFlags::EditorContext | EAutomationTestFlags::EngineFilter)
+
+bool FPLATEAUInstancedCityModelTest::RunTest(const FString& Parameters) {
```

Comment:
名前少し分かりづらいのでテスト名具体化お願いします！
```suggestion
IMPLEMENT_CUSTOM_SIMPLE_AUTOMATION_TEST(FPLATEAUCityModelLoader_LoadTest, FPLATEAUAutomationTestBase, "PLATEAUTest.CityModelLoader.Load",
                                        EAutomationTestFlags::EditorContext | EAutomationTestFlags::EngineFilter)

bool FPLATEAUCityModelLoader_LoadTest::RunTest(const FString& Parameters) {
```